db/seeds.rb

```diff
@@ -37,6 +37,11 @@ def audit_modified?(type, name, attributes = {})
 
 def create_filters(role, collection)
   collection.group_by(&:resource_type).each do |resource, permissions|
+    # check if the filter exists already
+    next if Filter.joins(:permissions).find_by("permissions.id" => permissions.map(&:id),
+                                               :role_id => role.id
```

Comment:
The only concern I have with that solution is that I would worry about other plugins calling this method thinking it's idempotent when it's not.